bigtable/client/CMakeLists.txt

```diff
@@ -66,6 +66,8 @@ add_library(bigtable_client
         internal/unary_client_utils.h
         idempotent_mutation_policy.h
         idempotent_mutation_policy.cc
+        polling_policy.h
```

Comment:
`polling_policy.h` and `polling_policy.cc` are already in this list, that is why the Bazel build is breaking, remove the duplicate entries (then regenerate the `.bzl` files and commit those too).